Project/Library/LibraryViewModel.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  BookViewModel.swift
+//  Project
+//
+//  Created by Emmanuel Velez Agudelo on 31/10/18.
+//  Copyright © 2018 Wolox. All rights reserved.
+//
+
+import Foundation
+import Core
+import ReactiveSwift
+import Result
+class LibraryViewModel {
+    //private var books : [Book] = []
+    private let _books = MutableProperty<[BookViewModel]>([])
+    public let books:Property<[BookViewModel]>
+    private var currentPage : Int  = 1
```

Comment:
This is just a suggestion, but the code is usually more readable when it's grouped by access type. You'll see what I mean when you have like a thousand attributes and you have to find a particular one. Also remember the syntax convention for the `:` in Swift (`cheese: Food`, for example). 

My suggestion:
```private let _books = MutableProperty<[BookViewModel]>([])
private let booksRepository : BooksRepository
private var currentPage : Int  = 1

public let books:Property<[BookViewModel]>

init(booksRepository : BooksRepository = NetworkingBootstrapper.shared.createBooksRepository() ) {
    books = Property(_books)
    self.booksRepository = booksRepository
}